contracts/RemixRewardUpgradable.sol

```diff
@@ -82,7 +104,7 @@ contract Remix is Initializable, ERC721Upgradeable, ERC721EnumerableUpgradeable,
         override
     {}
 
-    function addType (string calldata tokenType) public onlyRole(DEFAULT_ADMIN_ROLE) {
+    function addType (string calldata tokenType) public canSafeMint {
```

Comment:
would you like to add `canSafeMint` for `removeType` too?

Reply:
it's not necessary, at this point only owner can remove a type.